### DISCUSSION TITLE
Consider dropping `xtask` in favor of just, make or scripts

### DISCUSSION THREAD

**Discussion by nickgerace** [nickgerace/gfold#267] [Ideas]
Related to #265, we should consider using a task runner like [just](https://github.com/casey/just) and dropping the `xtask` sub-crate.

This isn't a big priority, but given `xtask`'s low discoverability and the proposal that #265, we have a chance to flatten the repository into one crate.

**Comment by nickgerace** [comment]
@uncenter wanted to tag you in this [as well](https://github.com/nickgerace/gfold/issues/265#issuecomment-2529482423) in case you had any thoughts!

**Comment by uncenter** [comment]
Sounds good to me, I'm a big fan of just scripts as well.

**Comment by nickgerace** [comment]
Closed by https://github.com/nickgerace/gfold/pull/271
